### PR TITLE
Add InventoryItem Client-side functionality

### PR DESCRIPTION
**Pull request by msus**
Add InventoryItem Client-side functionality

### CODE REVIEW

File: EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		if (quantity < 0)
+			throw new IllegalArgumentException("Quantity must be a non-negative number.");
+		
+		// Prepare a InventoryItemRequest to send to server.
+		InventoryItemRequest inventoryUpdateItemRequest = 
+				new InventoryItemRequest(item.getItemId(), quantity, ekrutLocation);
+		
+		// Sending InventoryItemRequest and receiving InventoryItemResponse.
+		InventoryItemResponse inventoryItemUpdateResponse = sendRequest(inventoryUpdateItemRequest);
+		if (inventoryItemUpdateResponse.getResultCode() == "OK") return;
```
Comment (almogkh) [inline EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java]: This is not how you compare strings
```java
strA.equals(strB)
```

File: EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		// Prepare a InventoryItemRequest to send to server.
+		InventoryItemRequest inventoryGetItemsRequest = 
+				new InventoryItemRequest(ekrutLocation);
+		
+		// Sending InventoryItemRequest and receiving InventoryItemResponse.
+		InventoryItemResponse inventoryGetItemsResponse = sendRequest(inventoryGetItemsRequest);
+		
+		// ResultCode is not "OK" meaning we encountered an error.
+		String resultCode = inventoryGetItemsResponse.getResultCode();
+		if (resultCode != "OK")
```
Comment (almogkh) [inline EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java]: Same thing here...

File: EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		// Prepare a InventoryItemRequest to send to server.
+		InventoryItemRequest inventoryUpdateItemThresholdRequest = 
+				new InventoryItemRequest(item.getItemId(), ekrutLocation, threshold);
+		
+		// Sending InventoryItemRequest and receiving InventoryItemResponse.
+		InventoryItemResponse inventoryUpdateItemThresholdResponse = sendRequest(inventoryUpdateItemThresholdRequest);
+		
+		// ResultCode is not "OK" meaning we encountered an error.
+		String resultCode = inventoryUpdateItemThresholdResponse.getResultCode();
+		if (resultCode != "OK")
```
Comment (almogkh) [inline EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java]: ...And here...

**Comment by msus** [comment]
Please check 2nd commit on this branch

**Pull request merged**